lib/slack_message.ml

```diff
@@ -147,9 +154,14 @@ let condense_file_changes files =
     in
     sprintf "modified %d files%s" (List.length files) (if prefix_path = "" then "" else sprintf " in `%s/`" prefix_path)
 
-let populate_commit ?(include_changes = true) repository (api_commit : api_commit) =
+let populate_commit ?(include_changes = true) match_github_user_to_slack_id repository (api_commit : api_commit) =
   let ({ sha; commit; author; files; _ } : api_commit) = api_commit in
-  let title = Slack.pp_api_commit api_commit in
+  let slack_mention =
+    match author with
```

Comment:
Option.map_default ? or is it done like this for readability ?

Reply:
I don't remember, but I suppose the current match statement would look cleaner since using `Option.map_default` would look something like this:
```
let slack_mention =
    author
    |> Option.map_default
        (fun gh_user -> gh_user |> match_github_user_to_slack_id |> Slack.format_slack_mention)
        ""
```